tests/unit/transport/test_clientinfo.py

```diff
@@ -0,0 +1,105 @@
+import pytest
+
+from globus_sdk import __version__
+from globus_sdk.transport._clientinfo import DefaultGlobusClientInfo, GlobusClientInfo
+
+
+def parse_clientinfo(header):
+    """
+    Sample parser.
+
+    Including this in the testsuite not only validates the mechanical implementation of
+    X-Globus-Client-Info, but also acts as a safety check that we've thought through the
```

Comment:
"X-Globus-Client-Info" here, "X-Globus-ClientInfo" elsewhere. This location reflects my preferred spelling.